CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - file source: `data_dir` now falls back to global `data_dir` option if not specified
 - aws_kinesis_streams: Added configurable partition keys
 - topology: Added ability to disable individual sink healthchecks
+- aws_cloudwatch_logs: Add dynamic group and stream creation
```

Comment:
Do we still need to do this manually with the conventional commits stuff? /cc @binarylogic

Reply:
Correct, we'll be autogenerating release changelogs from the commits, no need to maintain this manually.